app/experimenter/nimbus-ui/src/components/FormExperimentOverviewPartial/index.test.tsx

```diff
@@ -56,6 +78,18 @@ describe("FormExperimentOverviewPartial", () => {
     }
   };
 
+  const checkExistingForm = async (expected: Record<string, string>) => {
+    for (const [labelText, fieldValue] of [
+      ["Public name", expected.name],
+      ["Hypothesis", expected.hypothesis],
+      ["Application", expected.application],
+      ["Public Description", expected.publicDescription],
+    ]) {
+      const fieldName = screen.getByLabelText(labelText);
+      expect((fieldName as HTMLInputElement).value).toEqual(fieldValue);
```

Comment:
@LZoog this is where the failure occurs, with the "Application" field, triggered by the "with an existing experiment..." test.